app/views/readings/index.html.erb

```diff
@@ -1,29 +1,29 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('History') %></h2>
+<h2 class="heading"><%= t("History") %></h2>
```

Comment:
Internationalizing a page is not as simple as only replacing `ts` with `t`. The gist is that you need to use a locale key inside `t()` and then place the English text into the [locale file](https://github.com/otwcode/otwarchive/blob/master/config/locales/views/en.yml).

```suggestion
<h2 class="heading"><%= t(".page_heading") %></h2>
```
And in `config/locales/views/en.yml`:
```yml
readings:
  index:
    page_heading: History
```

See the [wiki page for I18n](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards) for more in-depth information and examples.